services/functional-tests/src/test/java/org/infinispan/online/service/caching/CustomCachingServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package org.infinispan.online.service.caching;
+
+
+import org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift;
+import org.arquillian.cube.requirement.ArquillianConditionalRunner;
+import org.infinispan.online.service.endpoint.HotRodTester;
+import org.infinispan.online.service.scaling.ScalingTester;
+import org.infinispan.online.service.utils.CommandLines;
+import org.infinispan.online.service.utils.DeploymentHelper;
+import org.infinispan.online.service.utils.ReadinessCheck;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+
+@RunWith(ArquillianConditionalRunner.class)
+@RequiresOpenshift
+public class CustomCachingServiceTest {
```

Comment:
It seems overkill to create a new test just for this. Is it possible to reuse `CachingServiceTest` with a Factory method that provides parameters for each test like we do in https://github.com/infinispan/infinispan? So in this case you would have two tests, one with num_owners=2 and the other with 3 with a generic method `should_have_replication_factor` and then check the configured "numOwners"?

The factory approach will also make it easy to test other startup parameter combinations as well as testing that changing any of those values does not break the caching service behaviour.